docs/fab_security.rst

```diff
@@ -0,0 +1,63 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+..    http://www.apache.org/licenses/LICENSE-2.0
+
+..  Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Airflow UI Security
+===================
+
+Security of Airflow Webserver UI is handled by Flask AppBuilder (FAB).
+Please read its related `security document <http://flask-appbuilder.readthedocs.io/en/latest/security.html>`_
+regarding its security model.
+
+Default Roles
+-------------
+Airflow ships with a set of roles by default: Admin, User, Op, Viewer, and Public.
+Only ``Admin`` users could configure/alter the permissions for other roles. But it is not recommended
+that ``Admin`` users alter these default roles in any way by removing
+or adding permissions to them as these roles will be re-synchronized to their original values.
+
+Admin
+"""""
+``Admin`` users have all possible rights, including granting or revoking rights from
+other users and altering other people's slices and dashboards.
+
+Public
+""""""
+``Public`` users (anonymous) don't have any rights.
+
+Viewer
+""""""
+``Viewer`` users have limited `viewer permissions <https://github.com/apache/airflow/blob/v1-10-stable/airflow/www_rbac/security.py#L77-L100>`_
```

Comment:
would prefer not using line ranges because as we make changes to `security.py` it may potentially invalidate this comment pretty soon. some other ideas are
(1) hard-code the permissions in this file
(2) refer to the global variables in `security.py`
(3) move the default security configs out of `security.py` into a separate module, and refer to the entire module.

Reply:
Or make this a doc comment and then include the code/fn doc via autoclass or similar?